libs/platform/form/src/renderers/default-form.renderer.ts

```diff
@@ -79,6 +79,52 @@ export class DefaultFormRenderer implements FormRenderer {
     };
   }
 
+  protected renderInput(
+    field: FormFieldDefinition,
+    value?: string | boolean
+  ): TemplateResult {
+    const { pattern, title } = this.fieldValidationPattern(field);
+    let inputType = field.type || field.attributes?.type;
+
+    switch (field.type) {
+      case FormFieldType.Number:
+        inputType = 'number';
+        break;
+      case FormFieldType.Password:
+        inputType = 'password';
+        break;
+      case FormFieldType.Color:
+        inputType = 'color';
+        break;
+      case FormFieldType.Boolean:
+        inputType = 'checkbox';
+        break;
+      case FormFieldType.Toggle:
+        inputType = 'radio';
+        break;
+    }
```

Comment:
Set exception for toggle (radio) and boolean (checkbox)

Reply:
But `field.attributes?.type` will be ignore in that case. I thought the specific attribute type should be preferred

Reply:
Not sure that I got your POV. 
`field.attributes?.type` will be ignored only in the case of `boolean` and `toggle` values which should be mapped to "checkbox" and "radio" because those input types aren't accepted.

Reply:
When you use `field.attributes?.type` you bring a very specific editor type into the mix. You could imagine that the type in the model is boolean, but you want have a custom editor that at then will resolve the boolean attribute. With this implementation we cannot achieve this, as the standard boolean editor is always used, regardless of a custom type.

Reply:
So, the `toggle` shouldn't be mapped to `radio` as well because there can be used custom editor that resolves `toggle`?